test/script/appveyor/install-redis.ps1

```diff
@@ -0,0 +1,14 @@
+Write-Host "Downloading Redis..." -ForegroundColor Cyan
+$redisRoot = "$env:SYSTEMDRIVE\Redis"
+$zipPath = "$($env:USERPROFILE)\redis-2.8.19.zip"
+(New-Object Net.WebClient).DownloadFile('https://github.com/MSOpenTech/redis/releases/download/win-2.8.19/redis-2.8.19.zip', $zipPath)
+7z x $zipPath -y -o"$redisRoot" | Out-Null
+del $zipPath
+
+Write-Host "Installing Redis as a Windows service..."
+& "$redisRoot\redis-server.exe" --service-install
+
+Write-Host "Starting Redis service..."
+& "$redisRoot\redis-server.exe" --service-start
```

Comment:
why no wait-loop here, similar to ES? Because redis start time is usually quick enough?

Reply:
yeah, Redis starts really fast whereas ES might take 30+ seconds. In reality I don't think we need the wait loop for ES either as it takes a while to run `npm install` and boot up the other services after anyway - in which time ES probably have started. But this might be a problem if anybody ever runs this locally on a Windows box where they have already run `npm install` before.